### PR TITLE
Homepage redesign header/footer changes

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-6767

### CODE REVIEW

**Comment by mloginov** [comment]
need merge with master

File: source/stylesheets/pages/_index.scss [hunk 1]
```diff
@@ -96,12 +238,18 @@ body.index .header-block {
...
+  .description-block {
+    padding-top: 0;
+
+  }
 
   @include breakpoint($break-tablet) {
     @include container(14 of 16);
     width: span(14 of 16);
-    margin: 0 span(1 of 16);
+    margin: 0 auto;//span(1 of 16);
```
Comment (mloginov) [inline source/stylesheets/pages/_index.scss]: remove commented code

File: source/stylesheets/pages/_index.scss [hunk 2]
```diff
@@ -111,15 +259,103 @@ body.index .header-block {
...
+    padding: 1 * $base-line-height 0;
 
     h2 {
-      font-size: 22px;
-      font-weight: 300;
-      text-transform: uppercase;
-      color: $light-gray;
+      font-size: 24px;
+      font-weight: 400;
+      //text-transform: uppercase;
```
Comment (mloginov) [inline source/stylesheets/pages/_index.scss]: remove

**Comment by denpira** [comment]
Done

**Comment by mloginov** [comment]
:+1:

**Comment by MaximKraev** [comment]
:+1:

**Pull request merged**